### PR TITLE
Fix some broken links

### PR DESCRIPTION
**Pull request by clareliguori**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jaypipes — APPROVED** [review]
Hey, thanks @clareliguori! :) We've moved repo locations a few times over the years and these typos are a result. Appreciate your keen eye!

File: docs/content/docs/contributor-docs/testing.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ is for the warmed caches case.
 #### Repeat for other services
 
 We have end-to-end tests for all services listed in the `DEVELOPER-PREVIEW`,
-`BETA` and `GA` release statuses in our [service listing](../services)
+`BETA` and `GA` release statuses in our [service listing](../../community/services)
```
Comment (vijtrip2) [inline docs/content/docs/contributor-docs/testing.md:232]:
```suggestion
`BETA` and `GA` release statuses in our [service listing](../community/services)
```

**Review by vijtrip2 — COMMENTED** [review]
Thank you very much for fixing these broken links! Much Appreciated!
/approve

**Pull request merged**